File: Renacer.Web/src/app/modelos/ratingEvento.ts
import { Evento } from "../servicios/evento.service";
import { Socio } from "../servicios/socio.service";

export class Rating {
    id: number;
    ratingEvento: number;
    ratingContenido: number;
    ratingEncargado: number;
    idEvento: number;
    evento:Evento;   

    idSocio: number; 
    socio:Socio;
    textoDescripcion: string;

}

File: Renacer.Nucleo/Entidades/RatingEvento.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Renacer.Nucleo.Entidades
{
    [Table("RatingEvento")]
    public class RatingEvento
    {
        public int id { get; set; }
        public int ratingEvento { get; set; }
        public int ratingContenido { get; set; }
        public int ratingEncargado { get; set; }
        public string textoDescripcion { get; set; }
        public int idEvento { get; set; }
        [ForeignKey("idEvento")]
        public Evento evento { get; set; }
        public int idSocio { get; set; }
        [ForeignKey("idSocio")]
        public Socio socio { get; set; }

  



    }
}
